pytket/binders/passes.cpp

```diff
@@ -30,34 +31,27 @@ using json = nlohmann::json;
 
 namespace tket {
 
-void update_routing_config(RoutingConfig &config, py::kwargs kwargs) {
-  if (kwargs.contains("swap_lookahead"))
-    config.depth_limit = py::cast<unsigned>(kwargs["swap_lookahead"]);
-  if (kwargs.contains("bridge_lookahead"))
-    config.distrib_limit = py::cast<unsigned>(kwargs["bridge_lookahead"]);
-  if (kwargs.contains("bridge_interactions"))
-    config.interactions_limit =
-        py::cast<unsigned>(kwargs["bridge_interactions"]);
-  if (kwargs.contains("bridge_exponent"))
-    config.distrib_exponent = py::cast<unsigned>(kwargs["bridge_exponent"]);
-}
 static PassPtr gen_cx_mapping_pass_kwargs(
     const Architecture &arc, const PlacementPtr &placer, py::kwargs kwargs) {
-  RoutingConfig config = {};
-  update_routing_config(config, kwargs);
+  RoutingMethodPtr method = std::make_shared<LexiRouteRoutingMethod>(100);
```

Comment:
Ah yeah, this came up before and I only did standardisation in the Python. As I haven't finished complete benchmarking on my side it's hard to truly know the best default. By observation, in general lookahead rarely goes beyond ~5 depth. However at times the capacity to go much deeper is useful. As the frequency of it exploring the whole 100 is small, I'm going to change it to 100 as it will inhibit runtime in a minor way and cover most cases.